### PR TITLE
Add gen build wf

### PR DESCRIPTION
**Pull request by dimakis**

### CODE REVIEW

File: .github/workflows/generate_clients.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      uses: actions/checkout@v3
+    - name: Use Node.js 14
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'        
+    - name: Use Python 
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.9'
+    - name: Delete old generated giles
```
Comment (wtrocki) [inline .github/workflows/generate_clients.yaml:21]: Removing old giles? https://en.m.wikipedia.org/wiki/Giles_(surname)

Comment (dimakis) [reply .github/workflows/generate_clients.yaml:21]: Old Giles needs to go 😂

File: .github/workflows/update_openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          commit-message: >
+            chore(openapi): update ${{ env.CLIENT_ID }} OpenAPI document
+
+          author: "app-services-ci <app-services-ci@users.noreply.github.com>"
+          branch: chore/add-${{ env.CLIENT_ID }}-openapi
+          branch-suffix: timestamp
+          reviewers: '@redhat-developer/app-services-sdk-maintainers'
+          delete-branch: true
+          body: >
+            _This pull request was auto-generated_
```
Comment (wtrocki) [inline .github/workflows/update_openapi.yaml:48]: Is this instruction still valid

File: .github/workflows/generate_clients.yaml (hunk 1 above)
Comment (dimakis) [reply .github/workflows/generate_clients.yaml:21]: Great catch. The sdks that are there are not upto date. The poetry publish script runs and I can successfully create master sdk locally but I need to clean this up on Monday

**Pull request merged**